packages/scss/src/components/tag/vars.scss

```diff
@@ -6,4 +6,5 @@
 	--components-tag-lineHeight: var(--sizes-S-lineHeight);
 	--components-tag-decoration: none;
 	--components-tag-cursor: default;
+	--components-tag-iconBottom: 1px;
```

Comment:
I guess it's related to vertical alignement in some screens, but it seems weird on mine: 
![image](https://github.com/LuccaSA/lucca-front/assets/25581936/32ed9c1f-f238-4ed6-a9b4-f8f3a766383b)

Our icons are supposed to provide an alignement related to our grid / font sizes 🤔

Reply:
![Capture d’écran 2024-03-22 à 10 40 26](https://github.com/LuccaSA/lucca-front/assets/64789527/cca36e83-3fd2-4e38-adb1-1b3de7ad0c8a)

This is the result I get without.